src/utilities/setUp.js

```diff
@@ -0,0 +1,10 @@
+import { loglanFetch } from '../helpers/fetchData';
+
+const geolocation = require('geolocation');
```

Comment:
Use `import` and `export` instead of `require` or `module.exports`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))